tidb-lightning/tidb-lightning-physical-import-mode-usage.md

```diff
@@ -224,10 +224,6 @@ The [`num-threads`](/tikv-configuration-file.md#num-threads) configuration of Ti
 
 ## Configure disk quota <span class="version-mark">New in v6.2.0</span>
 
-> **Warning:**
```

Comment:
Same suggestion as the DM continuous validation feature.

Reply:
done